index.js

```diff
@@ -45,6 +47,15 @@ let supportLevel = (() => {
 	}
 
 	if (process.platform === 'win32') {
+		// First version of Node.js to include patch to libuv that enables colour
+		// output on Windows. Anything earlier and it won't work.
+		if (semver.lt(process.version, '7.5.0')) {
```

Comment:
This should just be a part of the `if` statement on line 56 and use `gte` instead.